homepage/views.py

```diff
@@ -77,7 +82,16 @@ def course(request, id):
     try:
         course = Course.objects.get(pk=id)
         reviews = Review.objects.filter(course=id).order_by('-likes_num')
-        return render(request, 'homepage/courses/course.html', {'id': id, 'course': course, 'reviews': reviews})
+        if(request.user.is_anonymous):
```

Comment:
Same ^^

Reply:
this would be too long (123 chars) and she would have to cut it midway, feels more readable like this

Reply:
@oriAdler same